Examples/Python/python/acts/examples/simulation.py

```diff
@@ -305,7 +298,7 @@ def addPythia8(
             )
         )
 
-    return evGen if returnEvGen else s
```

Comment:
As it says above, `evGen` was only returned for compatibility with `common.addPythia8`. Since that's been removed, we can drop this: `return s` (as I suggested before) or don't return anything.
Similarly, we can remove the docstring comments about `common.addPythia8`.